pkg/cli/admin/ocpcertificates/regeneratemco/rotatecerts.go

```diff
@@ -120,3 +124,38 @@ func (o *RegenerateMCOOptions) Run(ctx context.Context) error {
 	}
 	return nil
 }
+
+func getServerIPsFromInfra(cfg *configv1.Infrastructure) []string {
```

Comment:
It looks like we have [multiple copies of this logic](https://github.com/search?q=org%3Aopenshift%20%22PlatformStatus.BareMetal.APIServerInternalIP%22&type=code) around now; this would be the 3rd (the baremetal one is a different variant, but related and is a 4th thing).

Not a blocker but we should definitely get a helper function for this into openshift/api at least.

Reply:
> Not a blocker but we should definitely get a helper function for this into openshift/api at least.

openshift/api has no code.  But openshift/library-go, ok.